### DISCUSSION TITLE
The "ircv3" Module, differences between (v3) and (v4)

### DISCUSSION THREAD

**Discussion by GabrieleMax** [inspircd/inspircd#2141] [Support] [answered]
Hello everyone,

when I search o the inspircd's website the keyword "ircv3" I have two results:
1) https://docs.inspircd.org/3/modules/ircv3/
2) https://docs.inspircd.org/4/modules/ircv3/

I read both but it seems they are the same and they have the same features when you wrote the config files, maybe there are differences which I didn't catch! :)

**Comment by sadiepowell** [comment, marked as answer]
The functionality is the same it's just one of those docs pages is for v3 and one is for v4.
